main/update-wpt.js

```diff
@@ -0,0 +1,103 @@
+
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+'use strict';
+
+// Script hacked together to load raw data for exploration in a REPL.
+
+require('foam2');
+const path = require('path');
+const fs = require('fs');
+const URL = require('url').URL
+const flags = require('flags');
+
+require(path.resolve(__dirname, '..', 'data/raw.js'));
+require(path.resolve(__dirname, 'specs.js'));
```

Comment:
Could we just make it `data/specs.json` and use `fs.readFileSync(... JSON.parse(...))`?